### ISSUE TITLE
Error text contrast not sufficient on statement submit

### ISSUE THREAD

**Issue by brendanarnold** [DFE-Digital/polis-whitelabel#8]
**Expected behavior**:

Error text should have sufficient contrast for partially sighted users. Suggest foreground of `#FFFFFF` and background of `#A80000` instead. Include some padding e.g. 3px

**Actual behavior**:

Error text contrast not sufficient when submitting a statement

<img width="217" alt="Picture 1" src="https://user-images.githubusercontent.com/584658/205948974-7ee254a3-1974-4cca-ae81-3f08b4dcc368.png">

**To Reproduce**:

Enter too much text into the statement field when voting - you should see an error text appear

**Additional context**:

Look for other error fields in e.g. `client-admin` and the `login` fields
